packages/@lwc/engine-server/src/utils/elements.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+const VOID_ELEMENT_NAMES = new Set([
```

Comment:
why is this list different than the used in the [html parser](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/template-compiler/src/parser/utils/html-elements.ts#L140) ?

Reply:
Good point, I will extract this to the `@lwc/shared`.